ourjapanlife/findadoc-server
# chore: replace apollo-server with @apollo/server

Resolves #107 


# What changed
1. [apollo-server](https://www.npmjs.com/package/apollo-server) has been deprecated and replaced with [@apollo/server](https://www.npmjs.com/package/@apollo/server).
2. The [Get Started](https://www.apollographql.com/docs/apollo-server/getting-started) documentation for @apollo/server recommends using `await startStandaloneServer` to initialize the server, but some changes will need to be done to allow top-level `await` in our repo. For now, I settled for wrapping the await in an async function, but this should probably be refactored by someone more familiar with Typescript.
3. The **graphql-tag** package has been added, because `gql` is not part of @apollo/server.

# Testing instructions

1. Delete `node_modules`
2. Run `yarn install`
4. Run `yarn dev`
5. Confirm that GraphQL Studio is working
